addons/website_livechat/__manifest__.py

```diff
@@ -23,38 +23,32 @@
     'assets': {
         'web.assets_frontend': [
             'mail/static/src/js/utils.js',
-            'website_livechat/static/src/bugfix/public_bugfix.js',
             'im_livechat/static/src/legacy/public_livechat.js',
             'website_livechat/static/src/legacy/public_livechat.js',
             'im_livechat/static/src/legacy/public_livechat.scss',
-            'website_livechat/static/src/bugfix/public_bugfix.scss',
+            'website_livechat/static/src/components/website/website.scss'
```

Comment:
This is not yet a component, you should create a `legacy/public_livechat.scss` file in `website_livechat` like the others.